include/seqan/parallel/parallel_algorithms.h

```diff
@@ -140,9 +140,9 @@ sum(TSequence const &seq)
  * @see sum
  */
 
-template <typename TTarget, typename TSource, typename TParallelTag>
+template <typename TTarget, typename TSource>
 inline typename Value<TSource>::Type
-partialSum(TTarget &target, TSource const &source, Tag<TParallelTag> parallelTag)
+partialSum(TTarget &target, TSource const &source, Parallel)
```

Comment:
These changes would break the Serial implementation wouldn't they?
The thing is that the Splitter with the Serial tag would simply make it a single chunk leading to a sequential execution of the following for-loops. So that should not changed.